quasar/main.tf

```diff
@@ -208,3 +208,42 @@ resource "aws_security_group_rule" "rds-egress" {
   protocol          = "-1"                           # All protocols.
   cidr_blocks       = ["0.0.0.0/0"]
 }
+
+resource "aws_db_parameter_group" "quasar-qa" {
+  name   = "quasar"
+  family = "postgres10"
+
+  parameter {
+    name  = "rds_logical_replication"
+    value = "1"
+  }
+
+  parameter {
+    name  = "max_logical_replication_workers"
+    value = "500"
+  }
+}
+
+data "aws_ssm_parameter" "qa_username" {
+  name = "/quasar-qa/rds/username"
+}
+
+data "aws_ssm_parameter" "qa_password" {
+  name = "/quasar-qa/rds/password"
+}
+
+resource "aws_db_instance" "quasar-qa" {
```

Comment:
Do we need to set [`vpc_security_group_ids`](https://www.terraform.io/docs/providers/aws/r/db_instance.html#vpc_security_group_ids) on this instance with the security groups you created above?

Reply:
Security groups are for 🐸. All ports open all the time!

Good catch, and added it into config via latest commit.